server/dba/migrations/42-forests.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
       endDate: { type: Sequelize.DATE, field: 'end_date' },
       contact: { type: Sequelize.STRING(500), field: 'contact' },
       mapLinks: { type: Sequelize.STRING(500), field: 'map_links' },
-      woodCost: { type: Sequelize.NUMERIC(8, 2), field: 'wood_cost' },
+      woodCost: { type: Sequelize.DOUBLE, field: 'wood_cost' },
```

Comment:
Just curious why we're using a `DOUBLE` here? We don't use a `DOUBLE` for any other data type in the application. (Same for line 24 below)

Reply:
@briandavidson -  had so much trouble getting those migration files to work, I was trying everything. But, I switched em back to NUMBERIC  and pushed it up.